CHANGELOG.md

```diff
@@ -482,7 +482,7 @@ BUG FIXES:
 
 BREAKING CHANGES:
 
-  * Retry now has a sane maximum default. Previous versions of Consul Template
+  * Retry now has a reasonable maximum default. Previous versions of Consul Template
```

Comment:
In line with my comment below... this use of 'reasonable' becomes optional but would still make sense.